setup.py

```diff
@@ -7,7 +7,7 @@
 DESCRIPTION = ('A very simple library for querying a Conceptpower REST API.')
 LICENSE = 'GNU GPL 3'
 URL = 'https://github.com/erickpeirson/Conceptpower-API'
-VERSION = '1.9'
+VERSION = '2.0'
```

Comment:
let's wait with increasing the version number until we're actually releasing a version

Reply:
Reverted back the version to <b>1.9</b> from <b>2.0</b>